src/adapters/elementToTree.js

```diff
@@ -0,0 +1,22 @@
+import flatten from 'lodash/flatten';
+
+export default function elementToTree(el) {
+  if (el === null || typeof el !== 'object') {
+    return el;
+  }
+  const { type, props } = el;
+  const { children } = props;
+  let rendered = null;
+  if (Array.isArray(children)) {
+    rendered = flatten(children, true).map(elementToTree);
+  } else if (children !== undefined) {
+    rendered = elementToTree(children);
+  }
+  return {
+    nodeType: typeof type === 'string' ? 'host' : 'class',
```

Comment:
i assume "host" means "DOM element" for web; i'm confused tho, why would a non-host be a "class"? SFCs aren't classes, and really `React.createClass` components aren't either.

ReactFourteenAdapter defines "function" also; should that be available here?